src/Matchers/Any.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pest\Matchers;
+
+final class Any
```

Comment:
Can you mark this call as internal for now?

Reply:
done